src/main/java/bio/terra/cloudfunctions/common/UnSupportedCloudEventTypeException.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.cloudfunctions.common;
+
+/** Thrown to indicate that the Cloud Event Type is not supported. */
+public final class UnSupportedCloudEventTypeException extends Exception {
```

Comment:
Do we need this class?  Why not just use `IllegalArgumentException`?

Reply:
Replaced `UnSupportedCloudEventTypeException` by `IllegalArgumentException`.